### PR TITLE
Add support for glob syntax in file check

### PR DESCRIPTION
**Pull request by vboulineau**
### What does this PR do?

Add support for glob syntax in compliance file check

### Describe your test plan

Define a compliance check similar to:

```
  - id: cis-docker-1.2.0-3.8
    scope:
      docker: true
      kubernetesNode: false
      kubernetesCluster: false
    resources:
      - file:
          path: /etc/docker/certs.d/*.crt
        condition: (file.permissions & 0444) == file.permissions
```

Change permissions of a targeted file, make sure check fails

### CODE REVIEW

File: pkg/compliance/checks/env/env.go [hunk 1]
```diff
@@ -29,5 +29,6 @@ type Configuration interface {
 	Hostname() string
 	EtcGroupPath() string
 	NormalizePath(path string) string
+	RelativePath(path string) string
```
Comment (xornivore) [inline pkg/compliance/checks/env/env.go:32]: We might want to add comments for all these here 😄 it's not evident what relative path means, normalize might be hard to understand as well. Maybe we should rename these to `NormalizeToHostRoot` and `RelativeToHostRoot`?

**Review by xornivore — APPROVED** [review]
Looks great! Just had a small comment about method naming.

File: pkg/compliance/checks/env/env.go (hunk 1 above)
Comment (vboulineau) [reply pkg/compliance/checks/env/env.go:32]: I wanted to clarify, but I was thinking these functions may also resolve variables when/if we introduce them.

Comment (xornivore) [reply pkg/compliance/checks/env/env.go:32]: As in expanding env vars? Yeah so then we'll probably have a set of options for that as an argument? I am just struggling a bit with `RelativePath` but `NormalizePath` is also somewhat confusing.

**Pull request merged**